doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -96,57 +96,112 @@ For Germany (DE) the country code is `0x4445`. Thus, the value for an unknown ft
 | **Value** | **Description** | **Service- Version** |
 |----------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------------------|
 | `0x4445000000000000` | "default value"<br />unknown payment type: automatic processing through the fiskaltrust.SecurityMechanisms settings is attempted. | 1.3-                |
-| `0x4445000000000000` | "unknown payment type for DE"<br />This is handled like a Start Receipt.                                                          | 1.3-                |
+| `0x4445000000000000` | "unknown payment type for DE"<br />This is handled like a pos-receipt.                                                          | 1.3-                |
```

Comment:
Is it on purpose that the `default value` and the `unkown ` In Austria we do have this for [ReceiptCases](https://github.com/fiskaltrust/interface-doc/blob/master/doc/appendix-at-rksv/appendix-at-rksv.md#type-of-receipt-ftreceiptcase) we do have this for the [PayItems](https://github.com/fiskaltrust/interface-doc/blob/master/doc/appendix-at-rksv/appendix-at-rksv.md#type-of-receipt-ftreceiptcase), but I actually don´t know if this is anyway wrong.

Reply:
"unknown payment type for DE"<br />This is handled like a pos-receipt.  
I suggest changing to

 "unknown receipt for DE"<br />This is handled like a pos-receipt and hence also ends a transaction

Reply:
i will clean up the numbers once we have all lines in place, then there will bony be one.